test/TestProjects/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.cs

```diff
@@ -0,0 +1,2056 @@
+        /// </description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
+        /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
+        /// <returns> The response returned from the service. </returns>
+        /// <include file="Docs/UnbrandedTypeSpecClient.xml" path="doc/members/member[@name='StillConvenient(RequestContext)']/*" />
+        internal virtual Result StillConvenient(RequestContext context = null)
+        {
+            using var scope = ClientDiagnostics.CreateSpan("UnbrandedTypeSpecClient.StillConvenient");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateStillConvenientRequest(context);
+                return _pipeline.ProcessMessage(message, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        internal HttpMessage CreateTopActionRequest(DateTimeOffset action, RequestContext context)
+        {
+            var message = new HttpMessage(_pipeline.CreateRestMessage(context, ResponseClassifier200));
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.Reset(_endpoint);
+            uri.AppendPath("/top/", false);
+            uri.AppendPath(action, "O", true);
+            uri.AppendQuery("api-version", _apiVersion, true);
+            request.Uri = uri;
+            request.Headers.Add("Accept", "application/json");
+            return message;
+        }
+
+        internal HttpMessage CreateTopAction2Request(RequestContext context)
+        {
+            var message = new HttpMessage(_pipeline.CreateRestMessage(context, ResponseClassifier200));
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.Reset(_endpoint);
+            uri.AppendPath("/top2", false);
+            uri.AppendQuery("api-version", _apiVersion, true);
+            request.Uri = uri;
+            request.Headers.Add("Accept", "application/json");
+            return message;
+        }
+
+        internal HttpMessage CreatePatchActionRequest(RequestContent content, RequestContext context)
+        {
+            var message = new HttpMessage(_pipeline.CreateRestMessage(context, ResponseClassifier200));
+            var request = message.Request;
+            request.Method = RequestMethod.Patch;
+            var uri = new RawRequestUriBuilder();
```

Comment:
```suggestion
            var uri = new RequestUri();
```

Reply:
Also, I couldn't find one here, but if you want to call 

`uri.AppendRaw(` on this, you need to do, e.g.:

`uri.AppendRawPathOrQueryOrHostOrScheme("/openai", false);`

For example:

```csharp
       internal RestMessage CreateGetChatCompletionsRequest(string deploymentId, RequestBody content, PipelineOptions context)
        {
            var message = new HttpMessage(_pipeline.CreateRestMessage(context, ResponseClassifier200));
            var request = message.Request;
            request.Method = RequestMethod.Post;
            var uri = new RequestUri();
            uri.Reset(_endpoint);
            uri.AppendRawPathOrQueryOrHostOrScheme("/openai", false);
            uri.AppendPath("/deployments/", false);
            uri.AppendPath(deploymentId, true);
            uri.AppendPath("/chat/completions", false);
            uri.AppendQuery("api-version", _apiVersion, true);
            request.Uri = new RawRequestUri(uri);
            request.Headers.Add("Accept", "application/json");
            request.Headers.Add("Content-Type", "application/json");
            request.Content = new RequestBodyContent(content);
            return message;
        }
```